galasa-parent/build.gradle

```diff
@@ -21,7 +21,7 @@ subprojects {
 
     ext.isReleaseVersion = !version.endsWith("SNAPSHOT")
 
-    def mavenrepo = System.getenv('MAVEN_REPO') ?: "https://nexus.galasa.dev/repository/maven-development"
+    def mavenrepo = System.getenv('MAVEN_REPO') ?: "https://nexus.galasa.dev/repository/maven-grade"
```

Comment:
nope.   override in the jenkins folder config if required

Reply:
Reverted 👍